pyrightconfig.stricter.json

```diff
@@ -69,6 +69,7 @@
         "stubs/stripe",
         "stubs/tqdm",
         "stubs/ttkthemes",
+        "stubs/untangle",
```

Comment:
Doesn't look like we actually need to exclude this package from the stricter pyright settings — it's already fully annotated!

```suggestion
```